src/components/PasswordField.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import {passwordRegex} from '../config/Constants';
+
+
+class PasswordField extends React.Component {
+   state = {
+       value:''
+   }
+
+   forBlur = (message) => {
+      this.setState({errormessage:message})
+   }
+    
+   focusForEveryInputElement = (event) => {
+    this.setState({errormessage:''});
+    if(event.target.classList.contains("invalid")) {
+        this.setState({errormessage:''});
+    }
+   }
+
+   validationForPassword = () => {
+    if(!(passwordRegex.test(this.state.value))) {
+       this.forBlur("Password must be atleast of length 8");
+    }
+    else{
+        this.props.methodToUpdateStateValue(this.props.nameOfStateProperty, this.state.value);
+    }
+   }
+
+    onInputForPassword = (event) => {
+        this.setState({value:event.target.value});
+        if(!(passwordRegex.test(this.state.value))){
+            event.target.classList.add('invalid');
+        }
```

Comment:
@KalariaYash remove this event.target.classList.add('invalid') we dont need as we are using error msg